### ISSUE TITLE
Ev3 Print3rbot construction (instruction) issue

### ISSUE THREAD

**Issue by AnastasiaPMP** [cavenel/ev3-print3rbot#15]
http://www.ev3dev.org/projects/2015/05/06/EV3-Print3rbot/
Hi! It's an awesome robot, thank you so much for this project!!

I am currently trying to build it myself by your instructions down the link above.

The problem I encountered is that the instruction puts the gear that lifts up the mechanism on one side of the robot. While the robot on the video has its gear in the middle, and the whole 'front' of the robot is slightly rebuilt.

Unfortunately, I can't figure out how I can deploy those changes so all my version of the robo does is smuge the picture.

Could you please help?﻿

**Comment by cavenel** [comment]
Hi! Thank you for your nice comment :)
The robot on the video is slightly older than the one from the instructions. The one from the instructions should work better than the one from the video.
If the robot doesn't lift enough, it can be something you can fix in the code (see closed issue here: https://github.com/cavenel/ev3-print3rbot/issues/3)

pen_up in the code is here: https://github.com/cavenel/ev3-print3rbot/blob/master/writer.py#L82 and you can try to increase 40 to 60 or 80 and see if it lifts better.

Does it lift correctly if you turn the motor yourself?

**Comment by cavenel** [comment]
After seeing your video from https://github.com/cavenel/ev3-print3rbot/issues/16 I would say the main problem is an incorrect position of the small arm. Look carefuly page 6 of the instructions, something must be wrong! The small sphere on the arm should be blocked by the red blocker (grey on your robot).

**Comment by cavenel** [comment]
Can I close this issue?

**Comment by danielhoherd** [comment]
I say close the issue. I just build the robot this week (2017-04-09) and had no problems with the instructions and the added annotations.

**Comment by AgentConDier** [comment]
I am not sure if this deserves its own issue, so since there is already a (still) open issue with the instructions I'll post this here. It my also be possible that this is what caused #16, the arm extending very far outwards is rather similar to what happened when I first turned it on with the cables connected in the wrong order.
According to the instructions, the left motor connects to C and the right one to A, but if you do this, it messes up, but if you connect the left motor to A and the other one to C it works perfectly fine! Nothing you cant figure out, but it is confusing.
Also, why not connect the cables on the side their respective motors/sensors are on, without crossing them over? It is currently slightly counter-intuitive.
 -- hope this helps! ;-)

**Comment by cavenel** [comment]
I will check the order but it's just a question of inverting motors which should be doable by someone that is trying to build this robot :) So I'm closing the issue!
